core/startup.js

```diff
@@ -31,11 +30,6 @@ var selectedModes    = null,
     },
     checkShutdownCode = function(code) {
         if (code === StatusFlag.ShutdownShouldRestart) {
-            startNewPlatform();
+            exports.run();
         }
     };
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530929747/issues/source?bid=3282987&fileBranchId=3283570#l35)